vanilla/scss/styles.scss

```diff
@@ -70,3 +70,28 @@ li > p {
 .admonition-security-notice {
   padding-left: 1rem;
 }
+
+// Toc custom styles
```

Comment:
Would it make sense to do it in Vanilla to reduce custom styling here?

Reply:
Do you mean all the styles? I'm not sure if we should hide the first child by default as in other use cases we might need the first child.

Reply:
No, not all, just indentation.
But I guess it makes sense to do it here locally and open an issue in Vanilla to add it there as well not to hold this for too long.

Reply:
Well, with first item hidden and everything else nested it seems the indentation styles here are quite specific, so it won't make so much sense to do it in Vanilla (or at least it won't be that helpful anyway).

So we can keep it here as custom styles.